lib/tasks/search.rake

```diff
@@ -1,170 +1,36 @@
 require 'task_helpers'
 
 namespace :search do
-  def import_from_cursor(cursor, index, opts)
-    tot = cursor.count
-    cnt = 0
-    t = Time.now
-    index.import cursor, {:method => :paginate, :per_page => opts[:batch_size]} do |documents|
```

Comment:
1. Yes. If we can't rate limit the reindex, we can't reindex in prod*
2. I don't feel strongly about feedback, but it'd certainly be useful to determine that the process wasn't hung
3. Define "errors". If the error isn't important to the reindex (i.e. it's "ignorable"), then we can skip 'em. If the error pertains to failing to reindex or reindexing incorrectly, then we'd better have some way of debugging it.

*If we fixed it so forums would read from the mongo secondary, this wouldn't be strictly critical, but it'd still be an important lever to have